### ISSUE TITLE
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by sdkilbo** [blaxxun-boop/PassivePowers#20]
Getting this error before and after updating to 1.1.0. Thanks for your work btw!

It basically spams this message, and makes my character unable to be moved.

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
PassivePowers.BossPowerPatches.ShipValue (Ship ship, System.Type type) (at <f75036ba776c430fac4d1e62ec876e16>:0)
PassivePowers.BossPowerPatches+IncreaseSailingSpeed.UpdateWindIntensity (System.Single windIntensity, Ship ship) (at <f75036ba776c430fac4d1e62ec876e16>:0)
(wrapper dynamic-method) Ship.DMD<Ship::GetSailForce>(Ship,single,single)
Ship.CustomFixedUpdate (System.Single fixedDeltaTime) (at <186f1bc3e8634801b40e626890156b17>:0)
MonoUpdatersExtra.CustomFixedUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <186f1bc3e8634801b40e626890156b17>:0)
MonoUpdaters.FixedUpdate () (at <186f1bc3e8634801b40e626890156b17>:0)

[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
PassivePowers.BossPowerPatches.ShipValue (Ship ship, System.Type type) (at <f75036ba776c430fac4d1e62ec876e16>:0)
PassivePowers.BossPowerPatches+IncreaseSailingSpeed.UpdateWindIntensity (System.Single windIntensity, Ship ship) (at <f75036ba776c430fac4d1e62ec876e16>:0)
(wrapper dynamic-method) Ship.DMD<Ship::GetSailForce>(Ship,single,single)
Ship.CustomFixedUpdate (System.Single fixedDeltaTime) (at <186f1bc3e8634801b40e626890156b17>:0)
MonoUpdatersExtra.CustomFixedUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <186f1bc3e8634801b40e626890156b17>:0)
MonoUpdaters.FixedUpdate () (at <186f1bc3e8634801b40e626890156b17>:0)
```

Note: It only occurs when i get within range of a ship.

**Comment by Glitxed** [comment]
Same problem, not only it disables movement, it also locks the character at the end of any animation, and makes him slide through the terrain, and the player cannot die

**Comment by blaxxun-boop** [comment]
Fixed in v1.1.1, thanks.
